backend/kale/marshal/__init__.py

```diff
@@ -1,35 +1,23 @@
-#  Copyright 2019-2020 The Kale Authors
+# Copyright 2019-2020 The Kale Authors
 #
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-#       http://www.apache.org/licenses/LICENSE-2.0
+#      http://www.apache.org/licenses/LICENSE-2.0
 #
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
```

Comment:
I hadn't notice that there are two spaces here. Maybe we can leave this as it is here and update all the Copyright headers in a future PR?

Reply:
Sure, done!